src/main/metabase/examples.ts

```diff
@@ -40,6 +40,21 @@ function lines(...lines: string[]) {
   return lines.join('\n');
 }
 
+function distinctValues(field: Field): number | undefined {
+  return field.fingerprint.global['distinct-count'];
+}
+
+const DISTINCT_THRESHOLD = 10;
+
+const colors = ['#88BF4D', '#A989C5', '#EF8C8C', '#F9D45C', '#F2A86F', '#98D9D9', '#7172AD'];
```

Comment:
so this cycles some colors, so that each histogram/chart uses a single but distinct color, correct?

Are you sure it looks good and is worth the code added?